site/downloads.markdown

```diff
@@ -25,152 +22,70 @@ This page describes the installation of the Haskell toolchain, which consists of
-
-To install from official openSUSE:Leap repo, just run the install command:
-
-`sudo zypper in ghc cabal-install`
-
-For last stable version you can use the development openSUSE repository:
+##### <span style="text-decoration: underline">openSUSE</span>
 
+* official packages
+  - [GHC](https://software.opensuse.org/package/ghc?search_term=%22ghc%22)
+  - [cabal](https://software.opensuse.org/package/cabal-install)
 * [devel:languages:haskell repo](https://build.opensuse.org/project/show/devel:languages:haskell)
 
-Use this command to add repository to your system:
-
-`sudo zypper ar -f -p 90 https://download.opensuse.org/repositories/devel:/languages:/haskell/openSUSE_Leap_15.1/devel:languages:haskell.repo`
-
-##### <span style="text-decoration: underline">openSUSE Tumbleweed</span>
-
-*   Tumbleweed has last stable version of ghc and cabal install.
-
-To install from official openSUSE:Tumbleweed repo, just run the install command:
-
-`sudo zypper in ghc cabal-install`
-
 ##### <span style="text-decoration: underline">Gentoo</span>
 
-While the Haskell toolchain is available in the main gentoo repository, it is recommended
-to use the more up-to-date [gentoo-haskell](https://github.com/gentoo-haskell/gentoo-haskell) overlay. This can be done using layman:
-
-```
-sudo layman -a haskell
-sudo emerge --ask dev-lang/ghc dev-haskell/cabal-install
-```
-
-</div>
-
-#### Windows
-
-1. [Configure Chocolatey](https://chocolatey.org/install) on your machine
-2. At an elevated command prompt, run `choco install haskell-dev haskell-stack`, followed by `refreshenv`.
+* official packages
+  - [GHC](https://packages.gentoo.org/packages/dev-lang/ghc)
+  - [cabal](https://packages.gentoo.org/packages/dev-haskell/cabal)
+  - [stack](https://packages.gentoo.org/packages/dev-haskell/stack)
+* [3rd party repo](https://github.com/gentoo-haskell/gentoo-haskell)
 
-### Official bindists { #bindists }
-
-GHC bindists are binary packages built as part of the GHC release process and are guaranteed to have passed the test suite. The installation process is a bit manual, so this is meant for power users.
-
-The [GHC download page](https://www.haskell.org/ghc/download.html) gives an extensive overview of available options. Navigate to the version you seek and click on **Binary packages**. You may also visit [downloads.haskell.org/~ghc](https://downloads.haskell.org/~ghc/) for the complete list. Likewise, to install cabal-install manually visit [downloads.haskell.org/~cabal](https://downloads.haskell.org/~cabal/cabal-install-latest/). Stack binaries can be found on [github.com/commercialhaskell/stack](https://github.com/commercialhaskell/stack/releases).
-
-Haskell-language-server binaries can be found [here](https://github.com/haskell/haskell-language-server/releases/).
+##### <span style="text-decoration: underline">Arch Linux</span>
 
-### Building from source { #from-source }
+**DO NOT USE ARCH LINUX PROVIDED GHC PACKAGES. THEY ARE BROKEN.**
```

Comment:
It would be helpful to provide at least some context for statement. For example we could link to an external explanation, such as <https://dixonary.co.uk/blog/haskell/cabal-2020>.

Reply:
> It would be helpful to provide at least some context for statement. 

Countless users on IRC over the years.

Reply:
Oh, I didn't mean I want you to provide *me* with *justification*.  Sorry if it came across like that. I meant from the point of view of an Arch user, if they see that comment, they'll likely think "Hmm, that's strange, I wonder what's broken about them ...". We can help short circuit their question by providing some sort of additional information, such as "Arch's insistence on using dynamic linking means that its Haskell packages are very fragile and we suggest that you do not use them.".

Reply:
> "Hmm, that's strange, I wonder what's broken about them ..."

My hope is they will ask in their community or their packagers, which will likely spread the word that Haskellers discourage their broken packages. I doubt that will make them think, but hey.

Reply:
Also: https://stackoverflow.com/a/65644318/2976251